<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="VideoWidget" name="video_widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="tabletTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="playList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="actionplay"/>
    <addaction name="actionpause"/>
    <addaction name="actionskip_forward"/>
    <addaction name="actionskip_backward"/>
    <addaction name="actionnext"/>
    <addaction name="actionprevious"/>
   </widget>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionopenfile"/>
    <addaction name="actionopen_folder"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>Audio</string>
    </property>
    <addaction name="actionvolume_down"/>
    <addaction name="actionvolume_up"/>
    <addaction name="actionmute"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionclear_playlist"/>
   </widget>
   <addaction name="menuPlayback"/>
   <addaction name="menuMedia"/>
   <addaction name="menuAudio"/>
   <addaction name="menuPlaylist"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionplay"/>
   <addaction name="actionpause"/>
   <addaction name="separator"/>
   <addaction name="actionprevious"/>
   <addaction name="actionskip_backward"/>
   <addaction name="actionstop"/>
   <addaction name="actionskip_forward"/>
   <addaction name="actionnext"/>
   <addaction name="separator"/>
   <addaction name="actionLoopOverAll"/>
   <addaction name="actionLoopOverSong"/>
   <addaction name="actionshuffle"/>
   <addaction name="separator"/>
   <addaction name="actionfullScreen"/>
   <addaction name="actionmute"/>
   <addaction name="actionvolume_down"/>
   <addaction name="actionvolume_up"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionopen_folder"/>
   <addaction name="actionopenfile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionclear_playlist"/>
  </widget>
  <action name="actionplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/play-circle.png</normaloff>:/src/icons/play-circle.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionpause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/pause-circle.png</normaloff>:/src/icons/pause-circle.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/skip-next.png</normaloff>:/src/icons/skip-next.png</iconset>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </action>
  <action name="actionprevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/skip-previous.png</normaloff>:/src/icons/skip-previous.png</iconset>
   </property>
   <property name="text">
    <string>previous</string>
   </property>
  </action>
  <action name="actionLoopOverSong">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/repeat-once.png</normaloff>:/src/icons/repeat-once.png</iconset>
   </property>
   <property name="text">
    <string>repeat one</string>
   </property>
  </action>
  <action name="actionshuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/shuffle-variant.png</normaloff>:/src/icons/shuffle-variant.png</iconset>
   </property>
   <property name="text">
    <string>shuffle</string>
   </property>
  </action>
  <action name="actionLoopOverAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/repeat.png</normaloff>:/src/icons/repeat.png</iconset>
   </property>
   <property name="text">
    <string>loop_over_all</string>
   </property>
  </action>
  <action name="actionvolume_up">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/volume-plus.png</normaloff>:/src/icons/volume-plus.png</iconset>
   </property>
   <property name="text">
    <string>volume up</string>
   </property>
  </action>
  <action name="actionvolume_down">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/volume-minus.png</normaloff>:/src/icons/volume-minus.png</iconset>
   </property>
   <property name="text">
    <string>volume down</string>
   </property>
  </action>
  <action name="actionmute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/volume-off.png</normaloff>:/src/icons/volume-off.png</iconset>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
  </action>
  <action name="actionstop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/stop-circle.png</normaloff>:/src/icons/stop-circle.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionopenfile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/open file.png</normaloff>:/src/icons/open file.png</iconset>
   </property>
   <property name="text">
    <string>open_file</string>
   </property>
  </action>
  <action name="actionfullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/fullscreen.png</normaloff>:/src/icons/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>fullScreen</string>
   </property>
  </action>
  <action name="actionopen_folder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/open folder.png</normaloff>:/src/icons/open folder.png</iconset>
   </property>
   <property name="text">
    <string>open_folder</string>
   </property>
  </action>
  <action name="actionskip_forward">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/skip-forward.png</normaloff>:/src/icons/skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>skip_forward</string>
   </property>
  </action>
  <action name="actionskip_backward">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/skip-backward.png</normaloff>:/src/icons/skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>skip_backward</string>
   </property>
  </action>
  <action name="actionclear_playlist">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/src/icons/4984950.png</normaloff>:/src/icons/4984950.png</iconset>
   </property>
   <property name="text">
    <string>clear playlist</string>
   </property>
   <property name="whatsThis">
    <string>clear playlist</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
